backend/local/local.go

```diff
@@ -1401,10 +1401,14 @@ func (o *Object) writeMetadata(metadata fs.Metadata) (err error) {
 
 func cleanRootPath(s string, noUNC bool, enc encoder.MultiEncoder) string {
 	if runtime.GOOS == "windows" {
-		if !filepath.IsAbs(s) && !strings.HasPrefix(s, "\\") {
-			s2, err := filepath.Abs(s)
-			if err == nil {
-				s = s2
+		if !strings.HasPrefix(s, "\\") {
```

Comment:
Read comment below first...

Maybe we should move this if statement above `if runtime.GOOS == "windows"` and make it work for all OSes

```go
if rutime.GOOS != "windows" || !strings.HasPrefix(s, "\\") {
```

Then we can avoid duplicating the following 8 lines of code

Note that `runtime.GOOS == "windows"` is evaluated entirely at compile time so effectively becomes a constant `true` or `false` so duplicating that is free!